config/application.rb

```diff
@@ -27,6 +27,10 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     config.i18n.default_locale = :ru
 
+    # Make public assets requireable in manifest files
+    config.assets.paths << Rails.root.join('public', 'assets', 'stylesheets')
+    config.assets.paths << Rails.root.join('public', 'assets', 'javascripts')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.